css/css-scoping/has-slotted-changing-001.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE HTML>
```

Comment:
Need to rename these files to `.tenatative`.

Reply:
Done and done.